docs/src/en/guide/get-started/README.md

```diff
@@ -28,7 +28,7 @@ Welcome to Waline. In just a few steps, you can enable Waline to provide comment
 
    ::: note
 
-   If you haven't logined, we recommend you to sign in with GitHub.
+   If you haven't logged ine, we recommend you to sign in with GitHub.
```

Comment:
I think there is a extra e

Reply:
There is, you can remove it. Thanks for flagging